docs/product-migration-guide/src/main/asciidoc/migration-upgrade-database-proc.adoc

```diff
@@ -0,0 +1,13 @@
+[id='migration-upgrade-database-proc_{context}']
+= Upgrade your database
+
+{PRODUCT} {PRODUCT_VERSION} contains updates to the database schema for optimal performance with the new {CENTRAL} and {KIE_SERVER}. You must update the {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} using one of the upgrade scripts provided with this release to accommodate the change.
+
+.Prerequisites
+* The *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons* distribution has been downloaded from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Red Hat Customer Portal] and extracted to a temporary directory.
+* The database has been backed up, as a precaution.
+
+.Procedure
+. Navigate to the extracted `{PRODUCT_FILE}-add-ons` folder in your temporary directory and extract the `{PRODUCT_INIT}-{PRODUCT_VERSION}-upgrade-scripts.zip` sub-folder. This folder contains database upgrade scripts that are required for applying changes to the database model made in {PRODUCT} {PRODUCT_VERSION}.
+. Log on to the {PRODUCT_OLD} 6.4 database to be used with {PRODUCT} {PRODUCT_VERSION} and run the relevant {PRODUCT} {PRODUCT_VERSION} upgrade script, according to the supported database type. This prepares the database for {PRODUCT} {PRODUCT_VERSION}.
```

Comment:
log on here but log in else where. IBM says both are fine, but choose one.

Reply:
It's intentional. Supposed to use log on for databases and the like, but log in otherwise.